$NetBSD$

  - Added the missing 'module_name' element in order to make creation of
    the .packlist possible. Module::Build relies on this element (at least
    0.2808 does), in addition to 'create_packlist'.

--- Build.PL.orig	2005-10-05 11:25:49.000000000 +0200
+++ Build.PL
@@ -123,6 +123,7 @@ my %xs = map { $subsystems{$_}{file}{fro
 	     keys %subsystems;
 
 my $build   = SDL::Build->new(
+	module_name          => 'SDL',
 	dist_name            => 'SDL_Perl', 
 	license		     => 'lgpl',
 	dist_version_from    => 'lib/SDL.pm',
